src/cosmos/useProvider.ts

```diff
@@ -12,15 +13,22 @@ enum Wallet {
   WalletConnect = 'WalletConnect',
 }
 const [metaMask] = initializeConnector<MetaMask>((actions) => new MetaMask({ actions }))
+
+const WALLET_CONNECT_PROJECT_ID = 'c6c9bacd35afa3eb9e6cccf6d8464395'
 const [walletConnect] = initializeConnector<WalletConnect>(
   (actions) =>
     new WalletConnect({
       actions,
       options: {
-        rpc: Object.entries(JSON_RPC_FALLBACK_ENDPOINTS).reduce((rpcMap, [chainId, rpcUrls]) => ({
+        rpcMap: Object.entries(JSON_RPC_FALLBACK_ENDPOINTS).reduce((rpcMap, [chainId, rpcUrls]) => ({
           ...rpcMap,
           [chainId]: rpcUrls.slice(0, 1),
         })),
+        showQrModal: true,
+        projectId: WALLET_CONNECT_PROJECT_ID,
+        chains: [SupportedChainId.MAINNET],
```

Comment:
This will refuse to connect to wallets which do not exist on MAINNET (eg a Safe that was created on an L2).
Can you leave that as a comment, so it's at least a known issue?